docs/spec_draft.md

```diff
@@ -1185,6 +1187,43 @@ output of `true_branch` is returned, else if pred is `false`, output of
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.imag
+
+### Semantics
+
+Extracts an imaginary value element-wise from `operand` and produces a `result`
+tensor.
```

Comment:
Fair question. I would keep the op simple since we will add tests that cover these and the reader can look at the example to figure it out. With the new wording proposed, it makes it more obvious that we are trying to get the imaginary part, and floats do not have it.

Reply:
Sounds good, keeping this comment open for reference for the next reviewer and approving.

Reply:
I think this should be documented. I wouldn't say that it's common knowledge that the imaginary part of a floating-point number is 0. This is an okay convention, but I think the common knowledge would be that "imaginary part" and "floating-point number" don't compute together. I've proposed a very simple formalization.